### PR TITLE
Add rust elastic-tube-1d solvers

### PR DESCRIPTION
**Pull request by fsimonis**
This PR adds rust variants of the elastic-tube-1d solvers.

To run, install preCICE and cargo (the Eierlegendewollmilchsau of rust), and finally run `./run.sh`.
Cargo will fetch dependencies, compile everything in release mode and run the solver for you.

### CODE REVIEW

File: elastic-tube-1d/README.md [hunk 1]
```diff
@@ -90,6 +90,22 @@ cd solid-python
...
+./run.sh
+```
+
+and
+
+```bash
+cd solid-rust
+./run.sh
+```
+
```
Comment (uekerman) [inline elastic-tube-1d/README.md:108]: Could be good to rewrite this section similar to other tutorials (e.g. https://precice.org/tutorials-perpendicular-flap.html#running-the-simulation). In the end, the participants are compatible with each other, right? Meaning `fluid-python` can be coupled to `solid-rust`.

Comment (fsimonis) [reply elastic-tube-1d/README.md:108]: Yes, they are compatible.

Comment (fsimonis) [reply elastic-tube-1d/README.md]: And done

File: elastic-tube-1d/fluid-rust/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+name = "fluid-rust"
+version = "0.1.0"
+edition = "2021"
+publish = false
+repository = "https://github.com/precice/tutorials"
+description = "The fluid participant of the elastic-tube 1D tutorial"
+license = "LGPL-3.0-or-later"
+
+[dependencies]
+"precice" = "2.5"
```
Comment (uekerman) [inline elastic-tube-1d/fluid-rust/Cargo.toml:11]: This tutorial requires preCICE v3 to run since the `precice-config.xml` requires it. How did you run it?

Comment (uekerman) [reply elastic-tube-1d/fluid-rust/Cargo.toml:11]: Tutorials are as much as possible in synch with preCICE. This one also requires the `develop` branch, e.g.

https://github.com/precice/tutorials/blob/5839e2cfa0f1577569f990a3a7845512b9c6b5c9/elastic-tube-1d/precice-config.xml#L22-L23

**Comment by fsimonis** [comment]
This PR is superseeded by #435

**Pull request closed without merging**